index.html

```diff
@@ -195,6 +195,32 @@ <h1>Skills</h1>
                 </div>
             </article>
         </section>
+        <section class="contact-form">
+            <article>
+                <div class="new-contact">
+                    <h1>Contact me</h1>
+                    <p id="new-info">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+                </div>
+                <div class="contact-fo">
+                    <form action="" method="post">
```

Comment:
- [ ] Kindly ensure that your form uses the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html)

Please open an account with [formspree service](https://formspree.io/html) so that all data entered in your form will be submitted there 👍🏽 .